refactor: Reference Transfer Form via public URL (#172)
citynetwork/docs: docs/howto/account-billing/delete-account.md

![Picture where you choose a option why you leave us. A field where you type your current password with a red delete button.](assets/reasonforleaving.png)

## Using the {{legal_docs.termination_form.name}}

> You may be unable to use the standard account deletion procedure using the {{gui}}, because you have lost your administrative credentials or you cannot access your account owner's email address.

Reviewer: Since there is a repetition of "use", and since I always find it somewhat strange to say (or even think) "use the procedure", I would suggest a slight re-phrase here.

```suggestion
You may be unable to follow the standard account deletion process using the {{gui}}, because you have lost your administrative credentials or you cannot access your account owner's email address.
```

Author: Sounds good.

In this case, you can instead terminate your account by filling out the [{{legal_docs.termination_form.name}}]({{legal_docs.termination_form.url}}).
Be sure to add your own signature (for personal accounts), or that of an authorized signatory (for company accounts).

Submit the form via our [{{support}}](https://{{support_domain}}/servicedesk).

> If you have lost your {{support}} credentials, you may also send it by email to [{{support_email}}](mailto:{{support_email}}).

Reviewer: ```suggestion
If you have lost your {{support}} credentials, you may also send it in by email to [{{support_email}}](mailto:{{support_email}}).
```

Author: Nah. :)

## What happens after you delete your account

> Three (3) days after sending the request for deletion, any running virtual machines will be shut off. Then, after ten (10) days, all virtual machines and any other resources connected to your account will be deleted, **including** backups.

Reviewer: I believe we are on the safe(r) side with the following ever-so-slight rephrasing:
```suggestion
Three (3) days after receiving the request for deletion, any running virtual machines will be shut off. Then, after ten (10) days, all virtual machines and any other resources connected to your account will be deleted, **including** backups.
```

Author: Fair!
